emr-launcher.tf

```diff
@@ -49,6 +78,30 @@ data "aws_iam_policy_document" "adg_emr_launcher_assume_policy" {
   }
 }
 
+data "aws_iam_policy_document" "adg_emr_launcher_read_s3_policy" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+    ]
+    resources = [
+      format("arn:aws:s3:::%s/emr/adg/*", data.terraform_remote_state.common.outputs.config_bucket.id)
+    ]
+  }
+}
+
+data "aws_iam_policy_document" "adg_emr_launcher_decrypt_s3_policy" {
+  statement {
```

Comment:
I'd be tempted to add this statement to the above policy doc as they're so closely related. That way you only have to attach a single policy to the role.

Reply:
Fixed by 1045c38